adr/0019-GPIO.md

```diff
@@ -0,0 +1,31 @@
+# 0019. GPIO
+
+Date: 2021-12-14
+
+## Status
+
+Accepted
+
+## Context
+
+We currently have a lot of integrations that use or are based on GPIO (including SPI and i2c busses).
+
+- All of these integrations have a low usage count (source Home Assistant Analytics),
+  but it adds the same amount of maintenance and review time to the core.
```

Comment:
```suggestion
  but add the same amount of maintenance and review time to the core.
```

Reply:
`it adds` or `adds`, but just `add` is incorrect in this case :)

Reply:
Take out the part before the comma and you have "All of these integrations ... add the same amount of ..."

I kinda see the "it" but to me it's confusing as the sentence started with a plural subject but seemed to flipped to singular. Alternatively, something like "...to support these integrations"  on the end of the sentence works and looking at it now I think that's the point?

Reply:
You are right ☺️ 
Thanks!